department-of-veterans-affairs/caseflow
# Ricky/appeals 27453

Resolves #{27453}
[APPEALS-27453](https://jira.devops.va.gov/browse/APPEALS-27453)

# Description
- Provide Legacy Appeals Seed Data for Demo environment for Remand Reason changes to Legacy.

## Acceptance Criteria
- [ ] Provides proper number of Legacy Appeals to use in Demo environment.

## Testing Plan

1. Go to #{27453} [APPEALS-27453](https://jira.devops.va.gov/browse/APPEALS-27453)
2. Check out the latest code
3. Run the Seed Script AdditionalLegacyRemandedAppeals
4. Ensure that the proper number of Legacy Appeals (50) is generated and are at the "Ready for Review" attorney workflow.

Expected Result:
1. 50 Legacy Appeals at the Ready for Review workflow is properly generated and operate normally.

# Frontend
## User Facing Changes
